lib/services/common.dart

```diff
@@ -4,7 +4,7 @@
 
 library dart_pad.common;
 
-final String serverURL = 'https://dart-services.appspot.com/';
+final String serverURL = 'http://doc-test.dart-services.appspot.com/';
```

Comment:
We can't do this in the main application code, sorry.

We can either use a cond-flag to turn this behaviour on, or we can wait for a day or so until I've finished stress testing the doc-test API and can make the version mainline.